src/main/java/org/opentripplanner/middleware/otp/response/LocalizedAlert.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.middleware.otp.response;
+
+import java.util.Date;
+
+public class LocalizedAlert {
+    public String alertHeaderText;
+    public String alertDescriptionText;
+    public String alertUrl;
+    public Date effectiveStartDate;
+
+    public boolean equals(Object o) {
```

Comment:
A hashCode can also be generated here to help facilitate checking these objects in a HashSet as recommended in https://github.com/ibi-group/otp-middleware/pull/45/files#r463329825.